tech.go

```diff
@@ -13,14 +14,19 @@ type Wappalyze struct {
 }
 
 // New creates a new tech detection instance
-func New() (*Wappalyze, error) {
+func New(customFilePaths ...string) (*Wappalyze, error) {
```

Comment:
How about we rename the `loadFingerprints` as `loadDefaultFingerprints` and then introduce a new method called something like `AddFingerprints` etc., to make it cleaner?